diff --git a/node_modules/@rbxts/chat-service/out/server.d.ts b/node_modules/@rbxts/chat-service/out/server.d.ts
index 6235eb5..56c8ce4 100644
--- a/node_modules/@rbxts/chat-service/out/server.d.ts
+++ b/node_modules/@rbxts/chat-service/out/server.d.ts
@@ -63,6 +63,10 @@ export interface ChatService {
      * Returns the speaker with the given name, or nil if it doesn’t exist.
      */
     GetSpeaker(speakerName: string): ChatSpeaker | undefined;
+    /**
+     * Returns all the current speakers in the game.
+     */
+    GetSpeakerList(): string[];
     /**
      * @deprecated
      * Undocumented
